redbeams/src/main/java/com/sequenceiq/redbeams/repository/DBStackRepository.java

```diff
@@ -1,15 +1,19 @@
 package com.sequenceiq.redbeams.repository;
 
-import com.sequenceiq.redbeams.domain.stack.DBStack;
-
 import java.util.Optional;
 
 import javax.transaction.Transactional;
 
 import org.springframework.data.jpa.repository.JpaRepository;
 
+import com.sequenceiq.cloudbreak.auth.altus.Crn;
+import com.sequenceiq.redbeams.domain.stack.DBStack;
+
 @Transactional(Transactional.TxType.REQUIRED)
 public interface DBStackRepository extends JpaRepository<DBStack, Long> {
 
-    Optional<DBStack> findByNameAndEnvironmentId(String name, String environmentId);
+    Optional<DBStack> findByNameAndEnvironmentCrn(String name, String environmentId);
```

Comment:
Does this work with the changed method name? I thought that JPA devised the query based on the method name. I could be wrong.

Reply:
In some of the entities (where there were no handcrafted queries) I did rename the field environmentId to environmentCrn, so this should work. 

On the other hand I am thinking we should retain environmentId, just to decouple the concept of an id from its representation as a crn. So I rename it back to environmentId. And, I think it would be better to rename this field for all or none of the entities but not just some.